CDP4Common/Poco/FileRevision.cs

```diff
@@ -40,14 +40,15 @@ protected string GetDerivedPath()
         {
             var path = new StringBuilder();
             var containingFolder = this.ContainingFolder;
+
             if (containingFolder != null)
             {
                 path.Append(containingFolder.Path);
                 path.Append("/");
+                path.Append(this.ContainingFolder.Name);
```

Comment:
is this not inlcuded in `containingFolder.Path` ?

Reply:
No, it's not.